docs/docsite/rst/shared_snippets/installing_collections_git_repo.txt

```diff
@@ -13,29 +18,26 @@ You can install a collection in a git repository by providing the URI to the rep
    ansible-galaxy collection install git@github.com:organization/repo_name.git
 
    # Install a collection from a local git repository
-   ansible-galaxy collection install git+file:///home/user/path/to/repo/.git
-
-In a ``requirements.yml`` file, you can also use the ``type`` and ``version`` keys in addition to using the ``git+repo,version`` syntax for the collection name.
-
-.. code-block:: yaml
-
-   collections:
-     - name: https://github.com/organization/repo_name.git
-       type: git
-       version: devel
+   ansible-galaxy collection install git+file:///home/user/path/to/repo_name.git
 
-Git repositories can be used for collection dependencies as well. This can be helpful for local development and testing but built/published artifacts should only have dependencies on other artifacts.
+.. warning::
 
-.. code-block:: yaml
+   Embedding credentials into a git URI is not secure. Use safe authentication options to prevent your credentials from being exposed in logs or elsewhere. 
+   
+   * Use `SSH <https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh>`_ authentication
+   * Use `netrc <https://linux.die.net/man/5/netrc>`_ authentication
+   * Use `http.extraHeader <https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpextraHeader>`_ in your git configuration
+   * Use `url.<base>.pushInsteadOf <https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtpushInsteadOf>`_ in your git configuration
 
-   dependencies: {'git@github.com:organization/repo_name.git': 'devel'}
+Specifying the collection location within the git repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Default repository search locations
------------------------------------
+When you install a collection from a git repository, Ansible uses the collection ``galaxy.yml`` or ``MANIFEST.json`` metadata file to build the collection. By default, Ansible searches two paths for collection ``galaxy.yml`` or ``MANIFEST.json`` metadata files:
 
-There are two paths searched in a repository for collections by default.
+* the top level of the repository
+* each directory in the repository path (one level deep)
```

Comment:
@acozine Capitalize the first word of each item in a vertical list (IBM Style Guide). I would also add a period at the end.

Reply:
Addressed in 60a5bfd89a256ebc871c5ecb87d7539c1c248c56. Thanks @lmalivert.